tests/fast-integration/skr-svcat-migration-test/skr-svcat-migration-test.js

```diff
@@ -104,6 +108,14 @@ describe("SKR SVCAT migration test", function() {
     await sampleResources.checkPodPresets(secretsAndPresets.podPresets, existing.podPresets)
   });
 
+  it(`Should restart functions pods`, async function() {
+    await t.restartFunctionsPods();
```

Comment:
this deletes the pods associated with functions `svcat-auditlog-api-1`, `svcat-auditlog-management-1` and `svcat-html5-apps-repo-1`, and waits for up to 100 seconds for the new pods to come to phase `Running`.